Qast/ViewControllers/MapViewController.swift

```diff
@@ -43,11 +30,10 @@ class MapViewController: UIViewController {
     var sensorDispatch = SensorDispatch(queue: .main)
     
     public let session: WearableDeviceSession
-    public let networker: SoundZoneAPI
     
-    init(session: WearableDeviceSession, networker: SoundZoneAPI = FirebaseManager()) {
+    init(session: WearableDeviceSession) {
         self.session = session
-        self.networker = networker
+        locationManager.getNearbySoundZones()
```

Comment:
this is gross: you have to call this in the LocationManager delegate's viewDidLoad to make it retrieve the SoundZones on start up.

Pretty sure there's a way to do this automatically in LocationManager's `init()`, but there are a lot of contingencies (can't use `self` in the networker's return closure in `init()`)

For now, I guess one line of boilerplate isn't the end of the world given the other goodies LocationManager provides